hivemind/hivemind_cli/run_dht.py

```diff
@@ -0,0 +1,76 @@
+import time
+from argparse import ArgumentParser
+
+from hivemind.dht import DHT, DHTNode
+from hivemind.utils.logging import get_logger, use_hivemind_log_handler
+from hivemind.utils.networking import log_visible_maddrs
+
+use_hivemind_log_handler("in_root_logger")
+logger = get_logger(__name__)
+
+
+async def report_status(dht: DHT, node: DHTNode):
+    logger.info(
+        f"{len(node.protocol.routing_table.uid_to_peer_id) + 1} DHT nodes (including this one) "
+        f"are in the local routing table "
+    )
+    logger.debug(f"Routing table contents: {node.protocol.routing_table}")
+    logger.info(f"Local storage contains {len(node.protocol.storage)} keys")
+    logger.debug(f"Local storage contents: {node.protocol.storage}")
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument(
```

Comment:
[Lots](https://github.com/learning-at-home/hivemind/blob/master/hivemind/p2p/p2p_daemon.py#L92) of other `P2P`/`DHT` params are not covered here, and we use some of them while testing libp2p connectivity features, etc.

What's your take on this? Should we add them gradually (when deemed necessary) or write code that automatically fetches their names, help lines, and defaults using reflexion?

In case of the former, I'm worried that the help lines & defaults would quickly become not synchronized.

Reply:
I guess that for the first PR that introduces `hivemind-dht`, we don't need to support all possible scenarios in which the DHT can be used. I deliberately kept it to a minimum to use the core functionality and took my inspiration from the ALBERT example; if you believe that some of the essential arguments are missing, I'd be happy to add them.

For the desynchronization of DHT docstrings, I'd not worry too much about it at first: I'm not aware of any imminent PRs that change the purpose of the given DHT arguments, and if somebody does change its API, I believe that at least one of us (me, you or @justheuristic) will have a look at the corresponding PR. Though it doesn't guarantee that we will notice a desync, the issue will be easily fixable, and adding extra abstraction at this point seems to me to be an overkill